app/assets/stylesheets/application.css.scss

```diff
@@ -168,3 +167,24 @@ h1 {
   text-overflow: ellipsis;
   font-size: 0.9em;
 }
+
+.select2-container .select2-choice {
+  background: none;
+  background-image: none;
+  height: 34px;
+  margin-right: 1px;
+}
+
+.select2-container .select2-choice > .select2-chosen {
+  margin-top: 2px;
+}
+
+.select2-container .select2-choice .select2-arrow {
```

Comment:
Merge rule `.select2-container .select2-choice .select2-arrow` with rule on line 171<br>Selector should have depth of applicability no greater than 2, but was 3